packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FC } from 'react';
+import { useCtraderServiceToken } from '@deriv/api';
+import { Button, Text } from '@deriv/quill-design';
+import { getPlatformFromUrl } from '../../../../../helpers/urls';
+import CTraderLabelIcon from '../../../../../public/images/ctrader-label.svg';
+import DerivXLabelIcon from '../../../../../public/images/derivx-label.svg';
+import LinuxIcon from '../../../../../public/images/ic-linux-logo.svg';
+import MacOSIcon from '../../../../../public/images/ic-macos-logo.svg';
+import MT5Icon from '../../../../../public/images/ic-mt5.svg';
+import WindowsIcon from '../../../../../public/images/ic-windows-logo.svg';
+import { THooks, TPlatforms } from '../../../../../types';
+import { PlatformDetails } from '../../../constants';
+
+type TAppContent = {
+    icon: JSX.Element;
+    link: string;
+    text: string;
+    title: string;
+};
+
+type AppToContentMapperType = {
+    [key: string]: TAppContent;
+};
+
+const AppToContentMapper: AppToContentMapperType = {
```

Comment:
Can we move it to new file like helper because it is a mapper Object
@thisyahlen-deriv

Reply:
Yes better to move it